src/engine-actions/create-refund-invoice.js

```diff
@@ -7,21 +7,21 @@ const { addInvoice, decodePaymentRequest } = require('../lnd-actions')
  * @type {String}
  * @default
  */
-const DEFAULT_INVOICE_EXPIRY = 31536000
+const DEFAULT_INVOICE_EXPIRY = '31536000'
```

Comment:
This seems like a big change. Just flagging to ensure you checked that the value is consumed as a string

Reply:
Yeah, the `expiry` of an invoice is an `int64` type in the `rpc.proto` so we would have to use a string to support the full range.